checkov/terraform/checks/resource/aws/RedshiftServerlessNamespaceKMSKey.py

```diff
@@ -0,0 +1,21 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+from checkov.common.models.consts import ANY_VALUE
+
+
+class RedshiftServerlessNamespaceKMSKey(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure that Redshift serverless namespace is encrypted by KMS"
+        id = "CKV_AWS_282"
+        supported_resources = ['aws_redshiftserverless_namespace']
+        categories = [CheckCategories.ENCRYPTION]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self) -> str:
+        return "kms_key_id"
+
+    def get_expected_value(self) -> str:
```

Comment:
```suggestion
    def get_expected_value(self) -> Any:
```
and don't forget to import `Any` 🙂